src/or/entrynodes.c

```diff
@@ -404,6 +404,17 @@ get_remove_unlisted_guards_after_days(void)
                                  DFLT_REMOVE_UNLISTED_GUARDS_AFTER_DAYS,
                                  1, 365*10);
 }
+
+/**
+ * If a guard has been unlisted for this many seconds continuously, we
```

Comment:
Usually we'll document a function by using a verb to describe what it does, like "Return the number of seconds after which we remove a guard that..."

Reply:
Reworded in be9401d703eadc675cf9782efd9a7750b4adfb2e.